crates/ruff_python_ast/src/comparable.rs

```diff
@@ -326,8 +326,14 @@ impl<'a> From<&'a ast::Constant> for ComparableConstant<'a> {
         match constant {
             ast::Constant::None => Self::None,
             ast::Constant::Bool(value) => Self::Bool(value),
-            ast::Constant::Str(value) => Self::Str(value),
-            ast::Constant::Bytes(value) => Self::Bytes(value),
+            ast::Constant::Str(ast::StringConstant {
+                value,
+                implicit_concatenated: _,
+            }) => Self::Str(value),
+            ast::Constant::Bytes(ast::BytesConstant {
+                value,
+                implicit_concatenated: _,
```

Comment:
Is the idea here that two strings compare equal regardless of whether they are implicitly concatenated? If not, add the `implicit_concatenated` flag to `Str` and `Bytes`.

Reply:
> Is the idea here that two strings compare equal regardless of whether they are implicitly concatenated?

Yes

```python
>>> "hello" " world" == "hello world"
True
```

Reply:
Yeah, that's intentional.

Reply:
(Adding a note.)